whelktool/src/main/groovy/datatool/scripts/mergeworks/Util.groovy

```diff
@@ -4,9 +4,13 @@ import org.apache.commons.lang3.StringUtils
 import whelk.Whelk
 import whelk.util.Unicode
 
-import java.util.regex.Pattern
-
 class Util {
+    static def whelk = Whelk.createLoadedCoreWhelk()
+
+    static def clusters = 'clusters'
+
+    static def contributionPath = ['@graph', 1, 'instanceOf', 'contribution']
```

Comment:
I'm not sure think we need these as constants for the scripts.
(And they should be uppercase in that case, and not follow my bad example :wink:)